pkg/imgpkg/cmd/push_test.go

```diff
@@ -116,26 +119,60 @@ func TestNestedImgpkgDirError(t *testing.T) {
 	}
 }
 
-func TestBundleFlagWithoutDirectoryError(t *testing.T) {
+func TestBundleDirectoryErrors(t *testing.T) {
 	tempDir := os.TempDir()
```

Comment:
I think we need to move this `os.TempDir` inside the `t.Run`, because if we parallelize the tests both tests would be running in the same folder at the same time, which could cause an intermittent test failure.  Or am I overlooking something? 
This will complicate a little bit the error message check 😢

Reply:
Argh. Check mate. You're right though. I'll figure something out wrt the error message.

Reply:
Maybe we don't need to include the value of `pushDir` as part of the path. After all, we mostly want to point out that at this juncture `images.yml` is missing, so I think outputting `.imgpkg/images.yml` is plenty good. I'll push a change with that and we can see how it looks.